generated/nidigital/nidigital/capture_waveform.py

```diff
@@ -0,0 +1,21 @@
+import collections.abc
+
+
+class CaptureWaveform(collections.abc.Sequence):
+    '''Used by fetch_capture_waveform() to return the capture waveform data'''
+    def __init__(self, capture_waveform_data):
+        self._capture_waveform_data = capture_waveform_data
```

Comment:
The class is already named "CaptureWaveform", so having the argument and attribute also have "capture waveform" in it is superfluous.